emoji-substitution/emojiMap.js

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
I split this file out because it could potentially get very lengthy and splitting it out improved readability and maintenance.